allennlp/data/token_indexers/pretrained_transformer_indexer.py

```diff
@@ -37,10 +39,13 @@ def __init__(
         super().__init__(token_min_padding_length)
         self._namespace = namespace
         self._tokenizer = AutoTokenizer.from_pretrained(model_name)
-        self._padding_value = self._tokenizer.convert_tokens_to_ids([self._tokenizer.pad_token])[0]
-        logger.info(f"Using token indexer padding value of {self._padding_value}")
         self._added_to_vocabulary = False
 
+        (
+            self._num_added_start_tokens,
```

Comment:
This is because they will very soon be used in this class when I submit the PR for the long sequence folding logic. If I put it in the subclass now, I'd have to move it here then. I'm fine with that though since you raised this point.